app/vue/src/server/iframe.html.js

```diff
@@ -29,7 +29,9 @@ const urlsFromAssets = assets => {
     .forEach(key => {
       let asset = assets[key];
       if (typeof asset === 'string') {
-        urls[getExtensionForFilename(asset)].push(asset);
+        if (Boolean(urls[getExtensionForFilename(asset)])) {
```

Comment:
This is fine but we usually just do `!!(urls[getExtensionForFilename(asset)])` to convert into boolean

Reply:
Oh, I copied it from few lines below. But actually it can be simplified to just `if (urls[getExtensionForFilename(asset)]) { ... }` - otherwise CircleCI test fails at `no-extra-boolean-cast` rule.